smart-metastore/src/main/java/org/smartdata/metastore/dao/AccessCountTableAggregator.java

```diff
@@ -36,18 +36,20 @@ public AccessCountTableAggregator(MetaStore metaStore) {
 
   public void aggregate(AccessCountTable destinationTable,
                         List<AccessCountTable> tablesToAggregate) throws MetaStoreException {
-    if (!tablesToAggregate.isEmpty()) {
-      ReentrantLock accessCountLock = metaStore.getAccessCountLock();
+    if (tablesToAggregate.isEmpty()) {
+      return;
+    }
+
+    ReentrantLock accessCountLock = metaStore.getAccessCountLock();
+    if (accessCountLock != null) {
```

Comment:
It turned out that the check was here because of several tests mocking `Metastore`. Mocked `getAccessCountLock()` method when needed and removed nullability check.